include/envoy/stats/symbol_table.h

```diff
@@ -158,6 +163,8 @@ class SymbolTable {
    * @param stat_name the stat name.
    */
   virtual void incRefCount(const StatName& stat_name) PURE;
+
+  virtual StoragePtr copyToBytes(absl::string_view name) PURE;
```

Comment:
nit: Add documentation here. It's not clear whether or not this function performs tokenization, bumps reference counting, etc.